pom.xml

```diff
@@ -73,9 +71,11 @@
         <license.maven.plugin.version>4.2</license.maven.plugin.version>
         <licensePath>LICENSE_HEADER_CPAL.txt</licensePath>
         <licenseYear>2023</licenseYear>
+        <commons.lang.version>3.13.0</commons.lang.version>
+        <mule.sdk.api.version>0.7.0</mule.sdk.api.version>
         <commons-dbcp2.version>2.9.0</commons-dbcp2.version>
         <mtf.javaopts></mtf.javaopts>
-        <mule.sdk.api.version>0.7.0</mule.sdk.api.version>
+        <awaitility.version>3.1.6</awaitility.version>
     </properties>
 
     <dependencyManagement>
```

Comment:
I don't know why it was there. Not that it's bad to have the dependencyManagement, it would even be useful if other projects used this as a parent, but maybe it makes it harder to read. I have changed it.